corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py

```diff
@@ -79,7 +79,7 @@ def migrate_from_migration(cls, apps, schema_editor):
             print("""
                 A migration must be performed before this environment can be upgraded to the latest version
                 of CommCareHQ. This migration is run using the management command {}.
-            """).format(command_name)
+            """.format(command_name))
```

Comment:
f-strings work on multi-line strings too, but github will not let me make a multi-line suggestion for changing this one.

Reply:
updated